src/gui/qgshighlight.h

```diff
@@ -125,6 +156,9 @@ class GUI_EXPORT QgsHighlight: public QgsMapCanvasItem
     void paintLine( QPainter *p, QgsPolylineXY line );
     void paintPolygon( QPainter *p, const QgsPolygonXY &polygon );
 
+    int mWidth = 1; // line / stroke width property
+    QColor mColor; // line / stroke color property
+    QColor mFillColor; // line / stroke fillColor property
```

Comment:
Are there no getters for these on pen and brush? Or are these performance considerations?

Reply:
What do you mean: forward the property to the brush?

Reply:
No, use `return mBrush->width();` in the getter.

Reply:
ok, got it, yes it should be possible to use the getters but maybe faster this way.

Reply:
Right, on the other hand I guess the setter is called more often (and time critical) and there it's additional overhead.
But it's just nitpicking really, I'm leaving the decision to you.